dom-event-delegation/main.js

```diff
@@ -0,0 +1,9 @@
+var $taskList = document.querySelector('.task-list');
+$taskList.addEventListener('click', clickTask);
+
+function clickTask(event) {
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+  console.log('closest .task-list-item', event.target.closest('.task-list-item'));
```

Comment:
make sure that you are checking to confirm that the element you clicked on was a button. That is a necessary check!